toast-tk-runtime/src/main/java/io/toast/tk/runtime/report/TemplateHelper.java

```diff
@@ -10,6 +15,11 @@
 
 public class TemplateHelper {
 
+	private static int Html_Length_Max = 40;
+	private static int Html_Length_Full = 100000;
+	//private static final Logger LOG = LogManager.getLogger(TemplateHelper.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)